README.md

```diff
@@ -12,16 +12,8 @@ A main loop is supplied that can repeatedly call a user function with a series o
 In order to utilise this, you can write your `app.py` as follows:
```

Comment:
Text above could do with some revision: 

```
 Purpose of this library to provide simple interface to parse HTTP 1.1 requests represented as string
```
While the FDK contract is HTTP, the intention is for that to be somewhat abstracted from the user - they write some Function code , this library helps them do that. 


```
 Handling Hot JSON Functions
 ---------------------------
````
Don't say Hot/cold any more   - I would say something like "Handling JSON input/output in functions"

Reply:
fixed